securitycenter/snippets/requirements.txt

```
@@ -1,2 +1,2 @@
-google-cloud-pubsub==2.21.5
+google-cloud-pubsub==2.28.0
```

Unrelated errors, remove this pin to unblock other tests
```suggestion
google-cloud-pubsub==2.21.5
```